presto-main/src/main/java/com/facebook/presto/operator/FileFragmentResultCacheManager.java

```diff
@@ -199,7 +203,8 @@ private static void tryDeleteFile(Path path)
             fragmentCacheStats.incrementCacheHit();
             return Optional.of(closeWhenExhausted(result, inputStream));
         }
-        catch (UncheckedIOException | IOException e) {
+        catch (Throwable e) {
+            log.error(e, "read path %s error", path);
```

Comment:
You don’t want to catch Throwable here- since “should be fatal” members of the Error subheirarchy like OutOfMemoryError are not what you want to catch here. Seems like catching Exception is sufficient here instead.

Reply:
Yes,I lack consideration.I hava fixed

Reply:
Should we consider only extending it to `RuntimeException`? I don't think we should cover checked exception here.

Also, does it make sense to catch this separately from `IOException` and `UncheckedIOException`? There are legit reasons for them to happen, but maybe not for other exceptions?

Reply:
thank you.